gitlab/resource_gitlab_pipeline_schedule.go

```diff
@@ -100,7 +99,9 @@ func resourceGitlabPipelineScheduleRead(d *schema.ResourceData, meta interface{}
 		}
 	}
 	if !found {
-		return errors.New(fmt.Sprintf("PipelineSchedule %d no longer exists in gitlab", pipelineScheduleID))
+		log.Printf("[DEBUG] PipelineSchedule %d no longer exists in gitlab", pipelineScheduleID)
```

Comment:
@roidelapluie 's original feedback on this was that we should only add auto-removal to resources that have an importer. Since this `gitlab_pipeline_schedule` resource doesn't have one, I think we should either add it or not have auto-removal.

Reply:
My own impression of his feedback was that it did not matter for "non critical" resources, which already have an importer. Of course this might be critical for some users. Not having a uniform behaviour across resources has been reported in #209 and is the root cause of the problem.

In any case, I'm not keen on adding an importer in that very same PR (let's try to get it shipped as soon as possible), so if you still intend on not autodeleting those resources from state, let me know it's fine by me and I'll remove that block!

Reply:
Would'nt a simple passthrough importer be enough?

Reply:
@roidelapluie maybe but I would have to add some tests I guess. I'll try something.

Reply:
I added importers and related tests/documentation